app/controllers/FileReceivedController.scala

```diff
@@ -51,13 +53,22 @@ class FileReceivedController @Inject() (
         fileDetails =>
           (getContactEmails, fileDetails) match {
             case (Some(emails), Some(details)) =>
-              val time = details.submitted.format(timeFormatter).toLowerCase
-              val date = details.submitted.format(dateFormatter)
-              for {
-                updatedAnswers <- Future.fromTry(request.userAnswers.remove(UploadIDPage))
-                _              <- sessionRepository.set(updatedAnswers)
-              } yield Ok(view(details.messageRefId, time, date, emails.firstContact, emails.secondContact))
-            case _ => Future.successful(InternalServerError(errorView()))
+              if (details.reportType.contains(SingleOther)) {
+                logger.warn("FileReceivedController: Test data submitted but successful outcome")
```

Comment:
I don't understand this warning is the clause only for test Data if it is SingleOther?

Reply:
My understanding is that SingleOther means that the report contains test data. Theoretically, the a SingleOther report should never get to this stage as it will have been caught earlier on in the journey. This is just a final catch that should never actually get triggered